packages/tomls/src/periphery/testnet.toml

```diff
@@ -1 +1,9 @@
-include = ["./mainnet.toml", "utils/deploy_proxy.toml"]
+include = [
+    "utils/deploy_proxy.toml",
+    "utils/upgrade_proxy.toml",
+    "configs/feature_flags.toml",
+    "configs/global_config.toml",
+    "configs/tokens-testnet/usdc.toml",
+    "configs/tokens-testnet/weth.toml",
+    "configs/tokens-testnet/wbtc.toml",
```

Comment:
no dai/usdt?

Reply:
hmm, good point. I should deploy them too.

Reply:
oh, I actually deploy them but not link them to periphery. As we want to be close to mainnet, I would leave them for later.